src/Classes/Controllers/CreateHiringPartnerContactController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Portal\Controllers;
+
+use Portal\Entities\HiringPartnerEntity;
+use Slim\Http\Request;
+use Slim\Http\Response;
+use Slim\Views\PhpRenderer;
+use Portal\Models\HiringPartnerModel;
+
+class CreateHiringPartnerContactController
+{
+    public $hiringPartnerModel;
+    public $renderer;
+
+    /**
+     * CreateHiringPartnerContactController constructor.
+     *
+     * @param $hiringPartnerModel
+     */
+    public function __construct(HiringPartnerModel $hiringPartnerModel)
```

Comment:
The model you used here is not the right one. There is another one called HiringPartnerContactsModel with the new functionalities.